### PR TITLE
keepassxc: update to 2.5.4

### PR DESCRIPTION
**Pull request by MasinAD**
* update to 2.5.4
* new patches

#### Description
Updated KeePassXC to 2.5.4 (current).

###### Type(s)
Update
- [x] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on
macOS 10.15.4 19E287
Xcode 11.4.1 11E503a 

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by cjones051073** [comment]
Azure tests are failing, please take a look.

**Comment by MasinAD** [comment]
I searched for Azure tests in regard to MacPorts but didn't find anything.
Could you please elaborate? What kind of tests do I have to make?

>

**Comment by mf2k** [comment]
Chris is saying that the builder failed and you should look into why. It appears from the error message that it failed to fetch. Could be a transient error. I forget how to tell it to try again. You could make a change to the Portfile however, I have a minor spacing issue you can fix anyway. I will submit that as a change request.

File: security/KeePassXC/Portfile [hunk 1]
```diff
@@ -20,19 +20,21 @@ platforms               darwin
...
-                        sha256  ef33258b859a7b996af007113613b0f6210f2341e8f5fb3a005564262c2caf30 \
-                        size    6798880 \
+                        rmd160  635c4eff7769c3edf80af1e3070d535157487ee6 \
+                        sha256  a55e0801c318b02b1ac4e16e9b7a87ccfa7b039ea60d2c62610bd1bbbdd6cd4a \
+                        size    6839396 \
                         ${distname}${extract.suffix}.sig \
-                        size    488
+                        rmd160  44153573ac36bc47b97e58b845bfee2a91db0c6a \
+                        sha256  fc527280cfa05c2ed75529c280432ab8d301f69911f2d74c56fe8061f2b0d2f1 \
+                        size 488
```
Comment (mf2k) [inline security/KeePassXC/Portfile:37]: Add some spaces to make '488' align with the other checksum values.

Comment (mf2k) [reply security/KeePassXC/Portfile:37]: This will also trigger a new rebuild attempt.

**Comment by cjones051073** [comment]
Please can you squash that last commit into the first one, such that there is a single commit in this PR

**Comment by MasinAD** [comment]
Done. Anything left to do? I'm new to this workflow :-)

File: security/KeePassXC/Portfile (hunk 1 above)
Comment (MasinAD) [reply security/KeePassXC/Portfile]: Done. And this time it worked.

**Pull request merged**